hoot-core/src/main/cpp/hoot/core/schema/OsmSchema.cpp

```diff
@@ -1765,6 +1765,7 @@ bool OsmSchema::isLinear(const Element &e)
     const Relation& r = dynamic_cast<const Relation&>(e);
     result |= r.getType() == MetadataTags::RelationMultilineString();
     result |= r.getType() == MetadataTags::RelationRoute();
+    result |= r.getType() == MetadataTags::RelationBoundary();
```

Comment:
This allows relation types of `boundary` to be written out and not dropped because they aren't recognized.